.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,8 +1,6 @@
 ---
 name: Bug Report
 description: Create a report to help us improve
-labels:
-  - bug
```

Comment:
This wasn't working to begin with because there is no `bug` label.

Rather than changing this to `type:bug` here, I opted to keep everything consistent and include it in the `.github/issue_labeler.yml` config